config/locales/self_service.en.yml

```diff
@@ -43,3 +43,7 @@ en:
       breadcrumb: "%{firm_name}"
       title: "Edit: %{firm_name}"
       saved: Saved successfully
+
+    onboarding:
+      title: Onboarding
```

Comment:
I'd be inclined to let the customer tell us whether or not to change this during the demo. It will draw attention to itself if it needs changed.